### DISCUSSION TITLE
Feature: Mail default to

### DISCUSSION THREAD

**Discussion by jimmitjoo** [laravel/framework#34509] [General]
I've just spent a few hours of debugging sent emails in my application as I was not aware of the "universal to" email. The docs explained this very clearly when I was reading it through. 

My initial expectation when I set the to-email in the config was that I had to use it myself, and not that it would override the emails set in the to method of the mail facade.

This brought a thought to me that it would be handy to have a "default to" address, as that was what I needed. This will only be used if there is no email address passed into the to method.

It seems like there are more people like me out there: https://laracasts.com/discuss/channels/laravel/default-to-address-in-mail

**Comment by ali-alharthi** [comment]
There's a trait called `Notifiable` and you can use it on models to send notifications (emails):
```
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Notifications\Notifiable;

class People extends Model
{
    use HasFactory;
    use Notifiable;
    ...
    protected $fillable = [
        ...
        'email',
        'personal_email',
        ...
    ];
    ...
    // Here we override the default column used by the trait (email) to (personal_email)
    public function routeNotificationForMail($notification)
    {
      return $this->personal_email;
    }
    ...
}
```
Then you can use it to send Notifications (with jobs (to queue), or anything really!):
```
...
use App\Models\People;
use App\Notifications\SomeNotification;
...
$people = People::all() // Every person
foreach($people as $person){
    $person->notify(new SomeNotification());
}
...
```
It is more dynamic and the proper way of sending. Making Laravel use a default to-email is not a good idea imo.
If you need it for testing you can certainly use tools such as [MailTrap](https://mailtrap.io).
- It does not matter what email(s) you send to, it will always be at your mailtrap box which a better way of testing imo. For example: test to see if email's content is being properly changed per user info - example: the name is printed properly for each user - ...etc!

**Reply by jimmitjoo** [reply]
You've probably misunderstood my suggestion. 

I am aware of notifications and mailtrap, but that does not change anything. The "to" part of mail config are overwriting notifications as well, the user that should have the emails is not getting it if the mail.to config is set. Then all notifications are going to that email instead.

**Reply by ali-alharthi** [reply]
Ok! I think, I got what you meant by default. Correct me if I'm wrong, but what are you suggesting is to have a default email just in case an email is not set within for example a notification?
If what I said is exactly what you meant, then I suggest building a package that adds that functionality - you never know it might be shipped w/ Laravel 9 by default ;).
